### PR TITLE
Add AC ORM

### PR DESCRIPTION
**Pull request by fernandohds564**
Guys, I generalized the code to measure BPMs noise and also the RF line of the response matrix.

@murilobalves , I also changed the LOCO package to consider the `bpm_weight` as a matrix, not a vector.
Please take a careful look to check whether I've done something wrong.

I've also added methods to save a loco_input file with all important infos for LOCO.
And also a method to save the response matrix to the ConfigDB Server.

### CODE REVIEW

File: apsuite/commisslib/meas_ac_orm.py [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+
+        self.sofb_data = SOFBFactory.create('SI')
+        if self.isonline:
+            self._create_devices()
+
+    @classmethod
+    def fitting_matrix(cls, tim, freqs, num_cycles):
+        """Create the matrix used for fitting of fourier components.
+
+        The ordering of the matriz is the following:
```
Comment (xresende) [inline apsuite/commisslib/meas_ac_orm.py:101]: typo `matriz`

**Review by xresende — APPROVED** [review]

File: apsuite/commisslib/meas_ac_orm.py (hunk 1 above)
Comment (fernandohds564) [reply apsuite/commisslib/meas_ac_orm.py]: thks! done!

**Review by anacso17 — APPROVED** [review]

File: apsuite/commisslib/meas_ac_orm.py [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+            for propty in propties_not_interested:
+                bpm.set_auto_monitor(propty, False)
+            for propty in propties_to_keep:
+                bpm.set_auto_monitor(propty, True)
+                pvo = bpm.pv_object(propty)
+                self._flags[pvo.pvname] = _Flag()
+                pvo.add_callback(self._set_flag)
+        self.devices.update(self.bpms)
+
+    def _do_measure(self):
```
Comment (murilobalves) [inline apsuite/commisslib/meas_ac_orm.py:415]: maybe add the measurement RF column here?

**Review by murilobalves — APPROVED** [review]

File: apsuite/commisslib/meas_ac_orm.py (hunk 2 above)
Comment (fernandohds564) [reply apsuite/commisslib/meas_ac_orm.py:415]: Right! I'll try to do it today, thanks!

**Review by vellosok75 — APPROVED** [review]

File: apsuite/commisslib/meas_ac_orm.py (hunk 2 above)
Comment (fernandohds564) [reply apsuite/commisslib/meas_ac_orm.py:452]: done!

**Review by xresende — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Pull request merged**